src/ApplyOfflineMapExe.cpp

```diff
@@ -81,7 +91,19 @@ int main(int argc, char** argv) {
 
 	// Log directory
 	std::string strLogDir;
-
+	
+	double lb;
+	
+	double ub;
+	
+	bool fCAAS;
```

Comment:
Please document use of fCAAS and fCAASLocal arguments, and what they preserve

Reply:
fCAAS is the CAAS algorithm with global bounds preservation.  The lower bound is "lb" and the upper bound is "ub."  fCAASLocal is the CAAS algorithm but with local bounds preservation.  Both preserve mass.